common/persistence/sql/sqlExecutionManager.go

```diff
@@ -1880,23 +1885,92 @@ func createTimerTasks(
 	return nil
 }
 
-func assertAndUpdateCurrentExecution(tx sqldb.Tx, shardID int, domainID sqldb.UUID, workflowID string, newRunID sqldb.UUID, previousRunID sqldb.UUID,
+func assertNotCurrentExecution(tx sqldb.Tx, shardID int, domainID sqldb.UUID, workflowID string, runID sqldb.UUID) error {
```

Comment:
I don't see this being used anywhere, do you need this ?

Reply:
this is from the 3+DC branch, probably leaked by rebase
i will remove this